Homebrew/Resolute Spellbook/firstdaybreak-additions.json

```diff
@@ -0,0 +1,366 @@
+        "concentration": false
+    },
+    {
+        "name": "Respite",
+        "level": 2,
+        "school": "Force",
+        "classes": "Light",
+        "subclasses": "",
+        "casttime": "1 bonus action",
+        "range": "60ft",
+        "components": {
+            "verbal": false,
+            "somatic": false,
+            "material": ""
+        },
+        "duration": "Instantaneous",
+        "ritual": false,
+        "description": "Choose one creature that you can see within range. The creature can immediately use its reaction to regain hit points equal to 2d6 + your forcecasting ability modifier. This power has no effect on droids or constructs.\n\n**Force Potency.** When you cast this power using a force slot of 3rd level or higher, the healing increases by 1d6 for each slot level above 1st.",
+        "concentration": false
+    },
+    {
+        "name": "Skill Proficiency",
+        "level": 2,
+        "school": "Force",
+        "automation": [
+            {
+                "type":"text",
+                "text": "Through the Force, you gain greater knowledge from those who came before you. Choose one skill in which you lack proficiency. For the power’s duration, you have proficiency in the chosen skill. The power ends early if you cast it again."
+            }
+        ],
+        "classes": "Universal",
+        "subclasses": "",
+        "casttime": "1 action",
+        "range": "Self",
+        "components": {
+            "verbal": false,
+            "somatic": false,
+            "material": ""
+        },
+        "duration": "1 hour",
+        "ritual": false,
+        "description": "Through the Force, you gain greater knowledge from those who came before you. Choose one skill in which you lack proficiency. For the power’s duration, you have proficiency in the chosen skill. The power ends early if you cast it again.",
+        "concentration": false
+    },
+    {
+        "name": "Transfer Force",
+        "level": 2,
+        "school": "Force",
+        "automation": [
+            {
+                "type": "roll",
+                "dice": "2d6 [necrotic]",
+                "name": "damage",
+                "higher": {
+                    "3": "1d6 [necrotic]",
+                    "4": "2d6 [necrotic]",
+                    "5": "3d6 [necrotic]",
+                    "6": "4d6 [necrotic]",
+                    "7": "5d6 [necrotic]",
+                    "8": "6d6 [necrotic]",
+                    "9": "7d6 [necrotic]"
+                }
+            },
+            {
+                "type": "roll",
+                "dice": "{caster.spellbook.spell_mod} [heal plus hit dice]",
+                "name": "heal",
+                "displayName": "Heal"
+            },
+            {
+                "type": "target",
+                "target": "each",
+                "effects": [
+                    {
+                        "type": "save",

```

Comment:
I would honestly just throw this in the text node as a variable there in the string concat